src/Console/Output/ProcessOutput.php

```diff
@@ -68,6 +85,20 @@ public function onFixerFileProcessed(FixerFileProcessedEvent $event)
     {
         $status = self::$eventStatusMap[$event->getStatus()];
         $this->output->write($this->output->isDecorated() ? sprintf($status['format'], $status['symbol']) : $status['symbol']);
+
+        ++$this->processedFiles;
+
+        $symbolsOnCurrentLine = $this->processedFiles % $this->symbolsPerLine;
+        $end = $this->processedFiles === $this->files;
+        if (0 === $symbolsOnCurrentLine || $end) {
+            $this->output->writeln(sprintf(
+                '%s %'.strlen((string) $this->files).'d / %d (%3d%%)',
```

Comment:
only downside I see is that is done in the constructor (76) already. I would remove it from the constructor and reuse the `strlen((string) $this->files)` in a var.